addons/l10n_be/data/account_chart_template_data.xml

```diff
@@ -7,6 +7,6 @@
             <field name="cash_account_code_prefix">570</field>
             <field name="transfer_account_code_prefix">580</field>
             <field name="currency_id" ref="base.EUR"/>
-            <field name="spoken_languages" eval="'nl_BE;nl_NL'"/>
+            <field name="spoken_languages" eval="'nl_BE;nl_NL;fr_FR;fr_BE;de_DE'"/>
```

Comment:
I don't know, there was already nl_NL so I added fr_FR as well. Does it matter?
This said, I don't think de_DE is an exaggeration. It is a national language after all...

Reply:
I think it is best to take the most specific ones:  nl_BE, fr_BE and de_DE  (I think there is some kind of inheritance nl_NL -> nl_BE @mart-e )

Reply:
This is used by `l10n_multilang`. When installing the chart of account, it will call [`process_coa_translations`](https://github.com/odoo/odoo/blob/f4ae831df40e803d1c4d7f9f39751b849b79021b/addons/l10n_multilang/models/l10n_multilang.py#L61) and copy the translations of the models `X.template` to the newly created `X` records. We added `nl_NL` and `nl_BE` because we want this translation mechanism to work with both. You don't need to add French as the source are already in French (and no translation is needed). You can add German if we have German translations (but I don't think it's the case).

Reply:
I made english as the base language and added french and german to the translations

Reply:
Ok then it's useful to add both fr_FR and fr_BE. But are you sure you want to have English as default? I though our policy was to use regional languages in chart of accounts. For instance, in China, the defaut is (was?) Mandarin and we had English translation (I say "was?" because I think there was a mess and not sure if still the case).

Reply:
I disagree with that policy. It is a mess to understand what is what when looking in the code (i.e. in `account.account.template.csv`) for people who don't speak the language. If english is one of the language we can have a translation for (here, I had everything with a script from xsd and xml files from the government), we should use english imo.

Reply:
Both approach have pros and cons, we can change it but just to be sure we are on the same page.